rakelib/build-test.rake

```diff
@@ -67,20 +66,21 @@ def xcode( scheme: '',
            reports_path: reports_path(),
            artifacts_path: artifacts_path()
           )
+  sh 'bundle exec pod install'
```

Comment:
This is not necessary. Should be done in `rake pod_install`.
Uncomment the line in the `rakelib/setup-project.rake`